src/main/java/org/zalando/fauxpas/FauxPas.java

```diff
@@ -56,4 +59,21 @@
         return predicate;
     }
 
+    public static <R> Function<Throwable, R> partially(
+            final ThrowingFunction<Throwable, R, Throwable> function) {
+        return throwable -> {
+            try {
+                return function.tryApply(unpack(throwable));
+            } catch (final RuntimeException e) {
```

Comment:
Why do you need this case?
Should it just always throw `CompletionException` for consistency?